packer/linux/scripts/install-docker.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-DOCKER_COMPOSE_V2_VERSION=2.24.6
-DOCKER_BUILDX_VERSION=0.13.0
+DOCKER_COMPOSE_V2_VERSION=2.27.0
```

Comment:
While I'm here, also bump Docker Compose from 2.24.6 to 2.27.0

Release notes:
- [2.24.7](https://github.com/docker/compose/releases/tag/v2.24.7)
- [2.25.0](https://github.com/docker/compose/releases/tag/v2.25.0)
- [2.26.0](https://github.com/docker/compose/releases/tag/v2.26.0)
- [2.26.1](https://github.com/docker/compose/releases/tag/v2.26.1)
- [2.27.0](https://github.com/docker/compose/releases/tag/v2.27.0)